docker/compose/mac-build.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2019 Intel Corporation
```

Comment:
Other Docker Compose files are at the root directory. And there are 2 (SGX and non-SGX). So this is not consistent with that convention.

Reply:
Ignore this--I thought this was TCF.